src/components/common/Navbar/Navbar.js

```diff
@@ -60,8 +60,8 @@ class Navbar extends Component {
     return (
       <Nav {...this.props}>
         <StyledContainer>
-          <LogoContainer>
-            <DevfolioLogo />
+          <LogoContainer aria-label="Devfolio Logo">
+            <DevfolioLogo aria-hidden="true" />
```

Comment:
Since the svg logo is purely decorative so I have hidden it from screen reader. The parent of it which is an anchor tag (which is tab-able as well) will provide the necessary info.

If you know a better way or if you feel it's not necessary then let me know